debezium-platform/postgresql-kafka-example/README.md

```diff
@@ -0,0 +1,197 @@
+Using Debezium-platfrom to manage and stream changes
+===
+This example  will walk you through on how to use the Debezium Management Platform to manage and stream changes from a PostgreSQL database into Apache Kafka.
+
+
+Preparing the Environment
+---
+As the first step we will provision a local Kubernetes cluster using [minikube](https://minikube.sigs.k8s.io/docs/) and will install an ingress controller. For this example, considering a local setup, we will use the `/etc/hosts` to resolve the domain.
+The following script, when executed, will use minikube to provision a local k8s cluster named `debezium` and will add the required ingress controllers. It will also update the `/etc/hosts` to add the domain url.
+
+```sh
+./create-environment.sh
+```
+> **_NOTE:_**
+If you are using minikube on Mac, you need also to run the `minikube tunnel -p debezium` command. For more details see [this](https://minikube.sigs.k8s.io/docs/drivers/docker/#known-issues) and [this](https://stackoverflow.com/questions/70961901/ingress-with-minikube-working-differently-on-mac-vs-ubuntu-when-to-set-etc-host).
+
+Now that you have the required k8s environment setup, its time to fire the required infra for this example. As we will be using PostgreSQL database and the Apache Kafka broker as source and the destination for our pipeline. The following script will create a dedicated namespace `debezium-platform` and use it going forward for further installations of our example. It will also provision the PostgreSQL database and the Apache Kafka broker.
+
+```shell
+./setup-infra.sh
+```
+
+you can check the required infra is up and running
+
+```shell
+
+$ kubectl get pods
+
+NAME                                        READY   STATUS    RESTARTS   AGE
+dbz-kafka-dual-role-0                       1/1     Running   0          98s
+dbz-kafka-entity-operator-9f4d8fbc4-twq7j   1/2     Running   0          14s
+postgresql-85cc668d48-pjn58                 1/1     Running   0          4m2s
+strimzi-cluster-operator-7dc6fbcbf5-h28dl   1/1     Running   0          3m59s
+
+```
+
+Deploying Debezium-platform Operator
+---
+We will install debezium-platfrom platform through helm 
+
+```shell
+helm repo add debezium https://charts.debezium.io &&
+helm install debezium-platform debezium/debezium-platform --version 3.1.0-beta1 --set database.enabled=true --set domain.url=platform.debezium.io
+
+```
+The domain.url is the only required property; it is used as host in the Ingress definition.
```

Comment:
Maybe would be better to clarify that `database.enabled=true` is used to deploy a PostgreSQl database used by the platform. If not you need to provide your own and provide the connection information. Like the note we have in [Installation](https://debezium.io/documentation//reference/stable/operations/debezium-platform.html#_installation)

Reply:
done